lib/components/checkbox/checkbox.style.dart

```diff
@@ -4,31 +4,30 @@ import 'package:remix_ui/components/checkbox/checkbox.variants.dart';
 
 class CheckboxStyles extends StyleRecipe<CheckboxStyles> {
   const CheckboxStyles({
-    required this.outerContainer,
+    required this.flexContainer,
```

Comment:
I believe `outerContainer` is a better description of the box itself. I do understand the problem you are trying to resolve.

Reply:
I renamed `outerContainer` to `flexContainer` to clarify that this style will be applied to a `FlexBox`.

Reply:
We should rename it back, but let's discuss how to create a Style type that can indicate that is a flex style.